includes/validation/class-amp-validation-error-taxonomy.php

```diff
@@ -1591,6 +1591,30 @@ public static function add_admin_notices() {
 		}
 	}
 
+	/**
+	 * Returns JSON-formatted error details for an error term.
+	 *
+	 * @param WP_Term $term The term.
+	 * @return string Encoded JSON.
+	 */
+	public static function get_error_details_json( $term ) {
+		$json = json_decode( $term->description, true );
+
+		// Convert the numeric constant value of the node_type to its constant name.
+		$xml_reader_reflection_class = new ReflectionClass( 'XMLReader' );
+		$constants                   = $xml_reader_reflection_class->getConstants();
+		foreach ( $constants as $key => $value ) {
+			if ( $json['node_type'] === $value ) {
+				$json['node_type'] = $key;
+				break;
+			}
+		}
+
+		$json['status'] = (bool) ( (int) $term->term_group & self::ACCEPTED_VALIDATION_ERROR_BIT_MASK ) ? __( 'Removed', 'amp' ) : __( 'Kept', 'amp' );
```

Comment:
I don't think we should use translations here. Also, we should capture the reviewed state. So I suggest:

```suggestion
		$json['removed']  = (bool) ( (int) $term->term_group & self::ACCEPTED_VALIDATION_ERROR_BIT_MASK );
		$json['reviewed'] = (bool) ( (int) $term->term_group & self::ACKNOWLEDGED_VALIDATION_ERROR_BIT_MASK );
```